addons/l10n_ro_edi_stock/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+{
+    'name': 'Romania - E-Transport',
+    'version': '1.0',
+    'category': 'Accounting/Localizations/EDI',
+    'description': """
+E-Transport implementation for Romania
+    """,
+    'depends': ['stock_delivery', 'sale_management', 'account_intrastat', 'l10n_ro_efactura'],
```

Comment:
I don't understand why we depend on `sale_management` & `account_intrastat` here

Reply:
`account_intrastat` is necessary because we need to check and send `intrastat_code_id` for some products. i just checked and `sale_management` is not needed, ill remove it